app-frontend/src/app/pages/market/model/model.controller.js

```diff
@@ -0,0 +1,94 @@
+export default class MarketModelController {
+    constructor( // eslint-disable-line max-params
+        $log, $state
+    ) {
+        'ngInject';
+        this.$log = $log;
+        this.$state = $state;
+
+        this.modelData = this.$state.params.modelData;
+        this.modelId = this.$state.params.id;
+        this.activeSlide = 0;
+        this.populateTestData();
+    }
+
+    populateTestData() {
+        this.similarQueryResult = {
+            count: 25,
+            hasNext: true,
+            hasPrevious: false,
+            pageSize: 10,
+            page: 1,
+            results: [{
+                name: 'Normalized Difference Vegetation Index - NVDI',
```

Comment:
Typo -- this should be `NDVI`; placeholders have a tendency to sneak into client demos so it's probably best to make sure it's correct.